From 82fb82d0ef9c1f8425864677963968951cc1cff1 Mon Sep 17 00:00:00 2001
From: Art_Chen <Chenxy0201@qq.com>
Date: Fri, 3 Nov 2023 20:31:21 +0800
Subject: [PATCH] arch: arm64: disable CONFIG_MODULE_SIG_PROTECT

This allowed us to load the modules that export the GKI protected
symbols.

Signed-off-by: Art_Chen <Chenxy0201@qq.com>
Change-Id: I2144b7a1144ca4a52a061c0e527813a348cbb4b1
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 29782a39fffa..b7ad20e1356d 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -90,7 +90,6 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 CONFIG_MODULE_SCMVERSION=y
 CONFIG_MODULE_SIG=y
-CONFIG_MODULE_SIG_PROTECT=y
 CONFIG_MODPROBE_PATH="/system/bin/modprobe"
 CONFIG_BLK_DEV_ZONED=y
 CONFIG_BLK_DEV_THROTTLING=y
-- 
2.42.0